core/domain/stats_services_test.py

```diff
@@ -14,20 +14,233 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+
 from core.domain import event_services
 from core.domain import exp_domain
 from core.domain import exp_services
-from core.domain import stats_jobs_continuous
 from core.domain import stats_domain
+from core.domain import stats_jobs_continuous
 from core.domain import stats_services
 from core.domain import user_services
 from core.platform import models
 from core.tests import test_utils
 import feconf
+import utils
 
 (stats_models,) = models.Registry.import_models([models.NAMES.statistics])
 
 
+class StatisticsServicesTest(test_utils.GenericTestBase):
+    """Test the helper functions and methods defined in the stats_services
+    module.
+    """
+    def setUp(self):
+        super(StatisticsServicesTest, self).setUp()
+        self.exp_id = 'exp_id1'
+        self.exp_version = 1
+        self.stats_model_id = (
+            stats_models.ExplorationStatsModel.create('exp_id1', 1, 0, 0, {}))
+
+    def test_calls_to_stats_methods(self):
+        """Test that calls are being made to the
+        handle_stats_creation_for_new_exp_version and
+        handle_stats_creation_for_new_exploration methods when an exploration is
+        created or updated.
```

Comment:
Using test_utils.CallCounter might be better here, to simplify the test.

Reply:
Done. This is better. Never knew about this. Thanks for pointing it out.